new/rfc-knife-bootstrap-client-config.md

```diff
@@ -0,0 +1,216 @@
+---
+RFC: unassigned
+Author: Jay Mundrawala <jdm@chef.io>
+Author: Bryan McLellan <btm@chef.io>
+Status: Draft
+Type: Standards Track
+
+---
+
+# Knife Boostrap `client.rb` Configuration
+
+Knife bootstrap currently only allows a select few `client.rb` parameters
+to be specified during bootstrap. This RFC is an attempt to allow users
+to be able to specify `client.rb` parameters that are not directly known
+to knife bootstrap.
+
+## Background
+
+The `knife-boostrap` command supports a small subset of possible parameters
+a user can tweak in the `client.rb`. It allows setting `node_name`,
+`verify_api_cert`, `ssl_verify_mode`, `http_proxy`, and `no_proxy`. The
+chef-client cookbook provides a template that supports many more, allowing
+the user to configure, during a Chef client converge, additional options
+such as disabled ohai plugins, start_handlers, report_handlers, exception
+handlers, etc. If something you are trying to do does not fit into the
+template provided, you are also free to specify config files in a `client.d`
+folder that will automatically be loaded.
```

Comment:
for the chef-client cookbook, I'd drop the list too. It's enough to convey it allows setting more configuration parameters, but not bulk/custom data, and the big thing is that it requires a "double-run" pattern where your first run won't have these parameters set.